pkgs/development/python-modules/hiredis/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, redis
+}:
+
+buildPythonPackage rec {
+  pname = "hiredis";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e97c953f08729900a5e740f1760305434d62db9f281ac351108d6c4b5bf51795";
+  };
+  propagatedBuildInputs = [ redis ];
+
+  # Test fail
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Python extension that wraps protocol parsing code in hiredis. It primarily speeds up parsing of multi bulk replies";
```

Comment:
This description is too long. Can we shorten it and not use a period at the end?

Reply:
fixed